airbyte-webapp/src/views/Connection/ConnectionForm/ConnectionForm.tsx

```diff
@@ -234,8 +238,8 @@ const ConnectionForm: React.FC<ConnectionFormProps> = ({
             </Section>
           )}
           <Section title={<FormattedMessage id="connection.transfer" />}>
-            <Field name="schedule">
-              {({ field, meta }: FieldProps<ConnectionSchedule>) => (
+            <Field name="scheduleData.basicSchedule">
```

Comment:
Does this rename introduce any issues? I feel like maybe?

Reply:
Hey could you help me with a example on what kind of issues?

Reply:
I believe the way formik saves data is based on the field name, so if we're referencing `.schedule` anywhere this could be an issue. I haven't dug into the code to check if that is the case however.

Reply:
Got it. Yeah makes sense. The other way around is to make a custom formik Schedule component (which has 2 dropdowns inside it) to handle this usecase. Does that sound like a reasonable implementation?

Happy to hear any other suggestions here

Reply:
I have tried to look for .schedule replaced them but yeah the question is quite valid